en/guide/development-tools.md

```diff
@@ -7,14 +7,15 @@ description: Nuxt.js helps you to make your web development enjoyable.
 
 ## End-to-End Testing
 
-[Ava](https://github.com/avajs/ava) is a powerful JavaScript testing framework, mixed with [jsdom](https://github.com/tmpvar/jsdom), we can use them to do end-to-end testing easily.
+[AVA](https://github.com/avajs/ava) is a powerful JavaScript testing framework, mixed with [jsdom](https://github.com/tmpvar/jsdom), we can use them to do end-to-end testing easily.
+
+First, we need to add AVA and jsdom as development dependencies:
 
-First, we need to add ava and jsdom as development dependencies:
 ```bash
 npm install --save-dev ava jsdom
 ```
 
-And add a test script to our `package.json` and configure ava to compile files that we import into our tests.
+Then add a test script to our `package.json` and configure AVA to compile files that we import into our tests.
```

Comment:
AVA in capitalize and « And » to « Then » to avoid repetition with « and » after package.